package.json

```diff
@@ -36,6 +36,7 @@
         "multer": "^1.4.2",
         "passport": "^0.4.0",
         "passport-local": "^1.0.0",
+        "promise.allsettled": "^1.0.2",
```

Comment:
where's this used?

Reply:
This is used in  `hacker.middleware.js` for `Promise.allSettled(...)`.